### PR TITLE
Add : comment and like functionality

### PR DESCRIPTION
**Pull request by b0llu**
## What to check
- [ ] src/Context/Post.context.js
- [ ] src/Pages/SinglePostPage/SinglePostPageComponents/CommentSection/CommentSection.jsx
- [ ] src/Pages/SinglePostPage/SinglePostPageComponents/SinglePost/SinglePost.jsx
- [ ] src/Reducer/ReducerCases.js

## What not to check
- [ ] Do not check for responsiveness
- [ ] Do not check any other files

live Preview,
https://stormmedia.netlify.app/

### CODE REVIEW

File: src/Pages/SinglePostPage/SinglePostPageComponents/CommentSection/CommentSection.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      {comments.length !== 0 &&
+        comments.map((commentData) => {
+          return (
+            <div key={uuid()} className={styles.comments}>
+              <div className={styles.post}>
+                <img
+                  src={
+                    userState.profilePhoto
+                      ? userState.profilePhoto
+                      : "https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png"
```
Comment (foolhardy21) [inline src/Pages/SinglePostPage/SinglePostPageComponents/CommentSection/CommentSection.jsx:48]: good use of ternary operator here.

File: src/Pages/SinglePostPage/SinglePostPageComponents/CommentSection/CommentSection.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+              addComment(comment);
+              setComment({ ...comment, comment: "" });
+            }}
+          >
+            Reply
+          </button>
+        </div>
+      </div>
+      {comments.length !== 0 &&
+        comments.map((commentData) => {
```
Comment (foolhardy21) [inline src/Pages/SinglePostPage/SinglePostPageComponents/CommentSection/CommentSection.jsx:40]: you can try optional chaining here `?.`, so you won't have to check the length seperately.

**Review by foolhardy21 — APPROVED** [review]
The logic and code seems good to me. nice work.

File: src/Components/EachPost/EachPost.jsx [hunk 3]
```diff
@@ -21,45 +26,75 @@ export const EachPost = () => {
...
+          <div className={styles.post_user_details}>
+            <div className={styles.user_name}>
+              <Link
+                style={{ color: "var(--content-color)" }}
+                to={`/${post.username}`}
+              >
+                <h1>{post.name}</h1>
+              </Link>
+              <Link
+                style={{ color: "var(--content-color)" }}
```
Comment (nayyyhaa) [inline src/Components/EachPost/EachPost.jsx:54]: Can remove the use of inline style from here.

**Review by nayyyhaa — APPROVED** [review]
Nice work. Your UI looks awesome 🥇 💯 code is also well structured with good use of ternary operator, use of constant action names.

File: src/Pages/SinglePostPage/SinglePostPageComponents/CommentSection/CommentSection.jsx (hunk 2 above)
Comment (b0llu) [reply src/Pages/SinglePostPage/SinglePostPageComponents/CommentSection/CommentSection.jsx:40]: Thanks for pointing it out i will check it out

File: src/Components/EachPost/EachPost.jsx (hunk 3 above)
Comment (b0llu) [reply src/Components/EachPost/EachPost.jsx:54]: yes i will remove the inline styling thanks for pointing it out

**Pull request merged**